test/src/test_slot.py

```diff
@@ -25,10 +25,10 @@ def _Run(self):
             return
 
         cmd = self.argv.mpi_cmd + " " + str(test.num_procs) + " "
-        cmd += self.argv.exe + " "
+        cmd += self.argv.exe + " -i "
         cmd += test.filename + " "
```

Comment:
This is cosmetic, but I would personally keep the flag and the value on the same line:
```
cmd += "-i " + test.filename + " "
```

Or even better (if you feel like it). Something like:
```
cmd = " ".join([ params ])
```

But maybe, we could change this in a different PR. Totally optional...

Reply:
I kept the flag and value on the same line. I'll defer the `join` another PR. There's quite a bit of cleanup that I think we could do there.